hw3/03-form.js

```diff
@@ -0,0 +1,68 @@
+const http = require("http");
+const express = require("express");
```

Comment:
I saw in the slack channel that, third party modules like express should not be used in HW3

Reply:
There's a Slack channel for this class?  What is it?

Reply:
Rewrote 03-form.js without Express.